src/docfx/build/legacy/LegacyItemType.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.Docs.Build
+{
+    internal enum LegacyItemType
+    {
+        [EnumMember(Value ="Toc")]
```

Comment:
```suggestion
        ~~[EnumMember(Value ="Toc")]~~
```

Reply:
Without this, the serialization is camel case